pkg/faketikv/cases/region_split.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cases
+
+import (
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/pd/server/core"
+	log "github.com/sirupsen/logrus"
+)
+
+func newRegionSplit() *Conf {
+	var conf Conf
+	for i := 1; i <= 3; i++ {
+		conf.Stores = append(conf.Stores, Store{
+			ID:        uint64(i),
+			Status:    metapb.StoreState_Up,
+			Capacity:  10 * gb,
+			Available: 9 * gb,
+		})
+	}
+	peers := []*metapb.Peer{
+		{Id: 4, StoreId: 1},
```

Comment:
why only one peer

Reply:
Split should work too when make up replica running at the same time. The simulate checker verifies regions on all stores have all split.